src/components/ChromeWindow.tsx

```
@@ -0,0 +1,451 @@
+      } ${closing ? "chrome-window-closing" : ""} ${
+        minimizing ? "chrome-window-minimizing" : ""
+      } ${restoring ? "chrome-window-restoring" : ""}`}
+      style={{
+        ...windowStyle,
+        zIndex,
+        borderRadius: maximized || snapped ? 0 : 8,
+        boxShadow: isActive
+          ? "0 12px 40px rgba(0, 0, 0, 0.5), 0 0 0 1px rgba(255, 255, 255, 0.08)"
+          : "0 4px 20px rgba(0, 0, 0, 0.35), 0 0 0 1px rgba(255, 255, 255, 0.04)",
+        opacity: 1,
+        transition: snapped && !dragRef.current.isDragging
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/components/ChromeWindow.tsx:346:33
  344 |           : "0 4px 20px rgba(0, 0, 0, 0.35), 0 0 0 1px rgba(255, 255, 255, 0.04)",
  345 |         opacity: 1,
> 346 |         transition: snapped && !dragRef.current.isDragging
      |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^ Cannot access ref value during render
  347 |           ? "left 0.2s ease-out, top 0.2s ease-out, width 0.2s ease-out, height 0.2s ease-out, opacity 0.15s, box-shadow 0.15s"
  348 |           : "opacity 0.15s, box-shadow 0.15s",
  349 |       }}

(react-hooks/refs)